src/Range/Pattern.php

```diff
@@ -77,7 +77,7 @@ public function __toString()
      */
     public static function fromString($range, $supportNonDecimalIPv4 = false)
     {
-        if (!is_string($range) || strpos($range, '*') === false) {
```

Comment:
Pattern ranges are in the form `1.2.3.*`, so if a string does not contain an asterisk, it's not a `Range\Pattern` range.